docs/source/examples/CAPI.ipynb

```diff
@@ -0,0 +1,474 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "root.plot(basis='yz', width=[2, 10], color_by='material', origin=[0., 0., 150.], pixels=[400, 400])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Settings: everything will be standard except for the temperature settings. Since we will be working with specified temperatures, you will need temperature dependent data. I typically use the endf data found here: https://openmc.org/official-data-libraries/\n",
+    "Make sure your cross sections environment variable is pointing to temperature-dependent data before using the following settings."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "lower_left = [-0.62992, -pitch/2, 0]\n",
+    "upper_right = [+0.62992, +pitch/2, +300]\n",
+    "uniform_dist = openmc.stats.Box(lower_left, upper_right, only_fissionable=True)\n",
+    "\n",
+    "settings_file = openmc.Settings()\n",
+    "settings_file.batches = 100\n",
+    "settings_file.inactive = 10\n",
+    "settings_file.particles = 200\n",
```

Comment:
This is really low for number of particles per batch. I would make it at least 1000 if not higher

Reply:
Sure thing! It's just an example, but I wouldn't want anyone to think 200 particles per batch was realistic for this geometry.